test/screenshot/spec/mdc-select/classes/baseline.html

```diff
@@ -77,14 +77,14 @@
                 Fruit
               </option>
             </select>
-            <label class="mdc-floating-label" for="my-select2">Pick a Food Group</label>
             <i class="mdc-select__dropdown-icon"></i>
             <div class="mdc-notched-outline">
-              <svg>
-                <path class="mdc-notched-outline__path"/>
-              </svg>
+              <div class="mdc-notched-outline__leading"></div>
+              <div class="mdc-notched-outline__notch">
+                <label class="mdc-floating-label" for="my-select2">Pick a Food Group</label>
```

Comment:
I've seen some instances where the floating label is still outside the notched outline element, and some where it's inside. Is that accidental or intentional?

Reply:
Accidental. The label should always be inside the notch if one exists.